CMakeLists.txt

```diff
@@ -104,6 +110,16 @@ install(TARGETS enet
     ARCHIVE DESTINATION ${CMAKE_INSTALL_LIBDIR}
     LIBRARY DESTINATION ${CMAKE_INSTALL_LIBDIR}
 )
-install(DIRECTORY ${CMAKE_CURRENT_SOURCE_DIR}/include/enet
```

Comment:
Why did you remove `${CMAKE_CURRENT_SOURCE_DIR}`?

Reply:
It's not necessary. Per the CMake documentation for `install()` (https://cmake.org/cmake/help/latest/command/install.html):

> The `FILES` form [i.e. `install(FILES...)` or `install(DIRECTORY...)`] specifies rules for installing files for a project. **File names given as relative paths are interpreted with respect to the current source directory.** Files installed by this form are by default given permissions `OWNER_WRITE`, `OWNER_READ`, `GROUP_READ`, and `WORLD_READ` if no `PERMISSIONS` argument is given.

Reply:
Thanks, this part looks good to me.
Don't have the knowledge to review the rest